controllers/component_build_controller_simple_build.go

```diff
@@ -66,7 +66,7 @@ func (r *ComponentBuildReconciler) SubmitNewBuild(ctx context.Context, component
 	buildPipelineRun, err := generatePipelineRunForComponent(component, pipelineRef, additionalPipelineParams, buildGitInfo)
 	if err != nil {
 		log.Error(err, fmt.Sprintf("Failed to generate PipelineRun to build %s component in %s namespace", component.Name, component.Namespace))
-		return err
+		return boerrors.NewBuildOpError(boerrors.EUnknownError, err)
```

Comment:
This change is unnecessary. The error returned from `generatePipelineRunForComponent` is already a persistent error `EInvalidDevfile`.

Reply:
ahh right that was recently added by Mykola, will remove this